### DISCUSSION TITLE
Sporadic Apps

### DISCUSSION THREAD

**Discussion by gj82854** [BOINC/boinc#5387] [General]
After reading the wiki discussion concerning this feature, I came away with the impression that there could possibly be worker to worker communication which I interpreted to mean client to client communication. I was under the understanding that one of the "security features" of the client was that the client always initiated the communication. In a sporadic app scenario, as I understand it, there could possibly be inbound communication to the client that was not initiated by the client. Is there any ideas about how this communication would be handle concerning firewall rules, network authentication, etc.? Of course, I may be completely misunderstanding the possible architecture.

**Comment by davidpanderson** [comment]
The peer-to-peer communication would be done by the app, not by the BOINC client.

**Comment by JeromeC2023** [comment]
Has any project expressed an interest in using this system so far?

**Comment by davidpanderson** [comment]
Yes, a couple.

**Reply by JeromeC2023** [reply]
Interesting, any more information about what project / their plans (if any at this stage) ?

**Comment by davidpanderson** [comment]
Distributed neural net training.  Planning stages AFAIK.
